packages/plugin-ethstorage/src/utils/blobs.ts

```diff
@@ -0,0 +1,137 @@
+const BlobTxBytesPerFieldElement: number = 32;      // Size in bytes of a field element
```

Comment:
Going forward, I think we should integrate the SDK instead of directly including the file when you have time.

Reply:
Adapting

Reply:
I think it may not be necessary for this PR